src/braket/pennylane_plugin/braket_device.py

```diff
@@ -111,7 +112,9 @@ def capabilities(cls):
     @property
     def operations(self) -> FrozenSet[str]:
         """FrozenSet[str]: The set of names of PennyLane operations that the device supports."""
-        return supported_operations()
+        if self._supported_ops is None:
+            self._supported_ops = supported_operations()
+        return self._supported_ops
```

Comment:
Why not just have `self._supported_ops = supported_operations()` in `__init__`? Then this just becomes
```python
return self._supported_ops
```

Reply:
Good point, updated! Thanks